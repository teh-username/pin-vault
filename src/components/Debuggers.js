import React from 'react';
import { List, ListItem } from 'react-native-elements';

const Debuggers = ({ handlePasscodeReset }) => (
  <List>
    <ListItem
      hideChevron
      title="Reset Passcode Settings"
      subtitle="Press if unable to enable Passcode Protection"
      onPress={handlePasscodeReset}
    />
  </List>
);

export default Debuggers;
