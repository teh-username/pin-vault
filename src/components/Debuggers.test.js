import React from 'react';
import Debuggers from './Debuggers';

describe('Debuggers component test', () => {
  it('renders as expected', () => {
    const wrapper = shallow(<Debuggers handlePasscodeReset={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
