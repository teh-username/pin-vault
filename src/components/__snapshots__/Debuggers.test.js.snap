// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Debuggers component test renders as expected 1`] = `
<List>
  <ListItem
    chevronColor="#bdc6cf"
    disabled={false}
    hideChevron={true}
    leftIconUnderlayColor="white"
    onPress={[Function]}
    rightIcon={
      Object {
        "name": "chevron-right",
      }
    }
    rightTitleNumberOfLines={1}
    roundAvatar={false}
    subtitle="Press if unable to enable Passcode Protection"
    subtitleNumberOfLines={1}
    switchButton={false}
    textInputEditable={true}
    title="Reset Passcode Settings"
    titleNumberOfLines={1}
    underlayColor="white"
  />
</List>
`;
